apps/console/src/features/roles/components/application-roles.tsx

```diff
@@ -89,15 +86,18 @@ export const ApplicationRoles: FunctionComponent<ApplicationRolesSettingsInterfa
 ): ReactElement => {
 
     const {
-        application,
         onUpdate,
         [ "data-componentid" ]: componentId
     } = props;
 
+    const path: string[] = history.location.pathname.split("/");
+    const appId: string = path[path.length - 1].split("#")[0];
```

Comment:
Can you check whether this would work?

```suggestion
    const appId: string = history.location.hash;
```

Reply:
Yeah it is working. It is a existing code.